src/core/client/admin/routes/Dashboard/CommentStatuses.tsx

```diff
@@ -0,0 +1,86 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import {
+  Bar,
+  BarChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+import { Environment } from "relay-runtime";
+
+import { CommentStatusesJSON } from "coral-common/rest/dashboard/types";
+import { createFetch, useFetch } from "coral-framework/lib/relay";
+
+import { CHART_COLOR_PRIMARY } from "./ChartColors";
+
+import styles from "./CommentStatuses.css";
+
+interface PieValue {
+  name: string;
+  value: number;
+}
+
+const CommentStatusesFetch = createFetch(
+  "CommentStatusesFetch",
+  async (environment: Environment, variables: any, { rest }) => {
+    const url = `/dashboard/${
+      variables.siteID ? variables.siteID + "/" : ""
+    }community-health`;
```

Comment:
Typically, when I see routes like this:

```
/dashboard/${siteID}/community-health

AND

/dashboard/community-health
```

I'd probably suggest instead using a query parameter to denote an optional filter:

```
/dashboard/community-health?siteID=${siteID}
```